### PR TITLE
Bump image tag to 1.4 & bump cct_module sprint to "sprint-27"'

### PR DESCRIPTION
**Pull request by slaskawi**

### CODE REVIEW

File: override-cp.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
-from: "redhat-sso-7/sso72:cp"
+from: "redhat-sso-7/sso72:latest"
```
Comment (iankko) [inline override-cp.yaml]: @slaskawi This is not needed (looks the `cp` tag on the standalone image is set based on `container.yaml` file depending in which branch the standalone image was built).

**Comment by iankko** [comment]
@slaskawi The other two changes (bump the cct_module version to `sprint-27` branch & image tag bump to `1.4` LGTM), but could change the commit message to reflect this? (That the image tag was bumped to 1.4 & cct_module sprint to `sprint-27`)

Otherwise LGTM (once these changes are done).

File: override-cp.yaml (hunk 1 above)
Comment (slaskawi) [reply override-cp.yaml]: Fixed, thanks!

**Review by iankko — APPROVED** [review]
LGTM, ACK.

**Pull request merged**